tests/code-gen-tests.rs

```diff
@@ -1,12 +1,12 @@
+#![cfg(feature = "experimental-code-gen")]
```

Comment:
🗺️ Similarly, I moved the feature gates in this file to the module level to avoid needing to repeat them.